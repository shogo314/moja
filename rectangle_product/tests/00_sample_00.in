5 5
0 0 2
0 3 3
1 2 5
1 4 7
3 1 11
0 0 5 5
0 0 5 4
0 2 5 5
1 1 3 5
1 0 3 2
