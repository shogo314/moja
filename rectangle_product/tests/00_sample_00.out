2320
330
105
35
1
